aws.yaml

```diff
@@ -301,9 +301,61 @@ kind: Release
 metadata:
   annotations:
     giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
-  name: v9.3.0
+  name: v9.3.1
 spec:
   state: active
+  date: 2020-05-12T11:00:00Z
+  apps:
+    - name: cert-exporter
+      version: 1.2.1
```

Comment:
Can we use 1.2.2? This has the Helm 3 change to use Release.Revision so we are prepared.

https://github.com/giantswarm/cert-exporter/tree/v1.2.2

Reply:
If its only helm3, I think it makes sense include in here.